.gitignore

```diff
@@ -10,10 +10,10 @@
 /monero-*
 
 # Generated binaries
-swapcli
-swapd
-swaprecover
-swaptester
+/swapcli
+/swapd
+/swaprecover
+/swaptester
```

Comment:
We only want to ignore the executables in the root folder and not the `cmd/*` folders that have the same names.